messenger/make_octave.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python
```

Comment:
I like the look of this file. I think `make.py` could be made to look like this after some cleanup, and then more duplication could be extracted out. But that doesn't have to be done now.